### PR TITLE
Add compatibility with hierarchicalMenu/breadcrumb/RatingsMenu in readme

### PR DESCRIPTION
**Pull request by bidoubiwa**
Per @gmourier tests, these widgets are compatible provided that you meet the requirements. 
For example the document structure and the settings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -606,15 +606,52 @@ instantsearch.widgets.refinementList({
 })
 ```
 
-### ❌ HierarchicalMenu
+### ✅ HierarchicalMenu
 
 [Hierarchical menu references](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/)
 
-The `hierarchicalMenu` widget is used to create a navigation based on a hierarchy of facet attributes. It is commonly used for categories with subcategories.
+The `hierarchicalMenu` widget is used to create a navigation based on a hierarchy of facet attributes. It is commonly used for categories with subcategories. See [usage](#hierarchical-menu-usage) bellow.
```
Comment (brunoocasali) [inline README.md:613]:
```suggestion
The `hierarchicalMenu` widget is used to create navigation based on a hierarchy of facet attributes. It is commonly used for categories with subcategories. See [usage](#hierarchical-menu-usage) below.
```

File: README.md [hunk 2]
```diff
@@ -606,15 +606,52 @@ instantsearch.widgets.refinementList({
...
 
-No compatibility because Meilisearch does not support hierarchical facets.
+- ✅ container: The CSS Selector or HTMLElement to insert the refinements. _required_
+- ✅ attribute: The name of the attributes to generate the menu with. _required_.
+- ✅ limit: How many facet values to retrieves.
```
Comment (brunoocasali) [inline README.md:617]:
```suggestion
- ✅ limit: How many facet values to retrieve.
```

File: README.md [hunk 3]
```diff
@@ -606,15 +606,52 @@ instantsearch.widgets.refinementList({
...
+- ✅ showMoreLimit: The maximum number of displayed items (min `2`).
+- ❌ separator: The level separator used in the records. (default `>`).
+- 🤷‍♀️ rootPath: The prefix path to use if the first level is not the root level.
+- ❌ showParentLevel: Whether to show the siblings of the selected parent level of the current refined value.
+- ✅ sortBy: How to sort refinements. [See guide](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/#widget-param-sortby)
+- ✅ templates: The templates to use for the widget.- ✅ templates: The templates to use for the widget.
+- ✅ cssClasses: The CSS classes to override.
+
+#### Hierarchical Menu Usage
+To make it work with Meilisearch your documents must have a specific structure, explaination on the structure can [be found here](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/#requirements).
```
Comment (brunoocasali) [inline README.md:628]:
```suggestion
To make it work with Meilisearch your documents must have a specific structure, an explanation of the structure can [be found here](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/#requirements).
```

File: README.md [hunk 4]
```diff
@@ -606,15 +606,52 @@ instantsearch.widgets.refinementList({
...
+
+Contrary to `instantsearch.js`, the hierchical fields are added in [`filterableAttributes`](https://docs.meilisearch.com/reference/api/filterable_attributes.html#update-filterable-attributes).
```
Comment (brunoocasali) [inline README.md:630]:
```suggestion
Contrary to `instantsearch.js`, the hierarchical fields are added in [`filterableAttributes`](https://docs.meilisearch.com/reference/api/filterable_attributes.html#update-filterable-attributes).
```

**Review by brunoocasali — CHANGES_REQUESTED** [review]
I found some typos regarding the English (grammarly did, actually 😝)

**Review by brunoocasali — APPROVED** [review]
LGTM! 🌮

**Comment by bidoubiwa** [comment]
bors merge

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**